Tutorials/Creating-Basic-Site/Document-Types/index-v8.md

```diff
@@ -0,0 +1,93 @@
+---
+versionFrom: 8.0.0
+---
+# Document Types
+
+## Data first 
+### nothing in = nothing out!
+
+Step 1 of any site is to create a "**_Document Type_**" – after a few installations you’ll become familiar with this terminology but at the start it might be a little bit bewildering.  A **_Document Type_** is a data container in Umbraco where you can add data fields/attributes where the editor user can input data and Umbraco can use it to output it in the relevant part of a "**_template_**" (more on these later).  
+
+**_Document Types_** are infinitely extendable but usually you’ll add data fields something like the following:
+
+*    Page title
+*    Sub Heading
+*    Body Text
+*    Meta Title
+*    Meta Description
+*    ...
+
+Each **_Data Field_** has a type - e.g. a text string or a number or rich text body... we’ll come to this later.
+
+## Creating your first Document Type
+
+Right, let’s get busy. Go to the **_Settings_** menu in Umbraco. This is the third button on the top menu. Then you’ll see a long list of settings – don’t worry about these yet, we’ll introduce them as we need them. 
+
+**_Document Types_** is positioned as the first option in the list and is always the starting point for any Umbraco build.  Hover over the **_Document Types_** **_node_** and you’ll see three dots **_..._** , click this to see the menu. Then you'll get four options, click the **_Document Type_** option - we want a template automatically created for us. Using folders can help you organise your Document Types but we'll keep things simple for now.
```

Comment:
Wouldn't it be neat to use a **note** for this part: "Using folders can help you organise your Document Types but we'll keep things simple for now." ?